node/tessel-export.js

```diff
@@ -916,6 +916,10 @@ Tessel.LED.prototype.toggle = function(callback) {
   this.write(this.value ? 0 : 1, callback);
 };
 
+Tessel.LED.prototype.isOn = function() {
+  return this.value === 1;
+};
+
```

Comment:
> That makes sense, but I don't really like the conflicting .on() thought

It's insane that LEDs were an EventEmitter to begin with, considering all operations on them are always known by all user land code—there is no external modifier (as there is with sensors) that need to broadcast their state change. Additionally, @johnnyman727 conceded that there was likely no code in the wild that used that API. There is most definitely code in the wild using the rest of the API, which is why it's being preserved. 

> I was thinking that value was the state of the led, and could only be off (0) or on (1). Maybe .state could be a better name? Or maybe .isOn is a good name, but than we could we get rid of .value?

Unfortunately, `.value` falls into the same category I described above. I would've preferred to stash it away in a WeakMap, but could not.